ci/test_wheel.sh

```diff
@@ -13,6 +13,21 @@ python -m pip install $(echo ./dist/cucim*.whl)[test]
 if [[ "$(arch)" == "aarch64" && ${RAPIDS_BUILD_TYPE} == "pull-request" ]]; then
     python ./ci/wheel_smoke_test.py
 else
+
+    DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends libopenslide-dev
```

Comment:
Do we need the dev package at test time or just the runtime library?

```suggestion
    DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends libopenslide
```

Reply:
Thanks, we shouldn't need the dev lib. Apprently the package is called `libopenslide0`, though